chapters/resources.txt

```diff
@@ -3689,6 +3689,24 @@ include::{generated}/api/enums/VkImageAspectFlagBits.txt[]
   * ename:VK_IMAGE_ASPECT_STENCIL_BIT specifies the stencil aspect.
   * ename:VK_IMAGE_ASPECT_METADATA_BIT specifies the metadata aspect, used
     for sparse <<sparsememory, sparse resource>> operations.
+ifdef::VK_VERSION_1_1,VK_KHR_sampler_ycbcr_conversion[]
+  * ename:VK_IMAGE_ASPECT_PLANE_0_BIT specifies the plane 0 of
+    _multi-planar_ image format.
+  * ename:VK_IMAGE_ASPECT_PLANE_1_BIT specifies the plane 1 of
+    _multi-planar_ image format.
+  * ename:VK_IMAGE_ASPECT_PLANE_2_BIT specifies the plane 2 of
+    _multi-planar_ image format.
+endif::VK_VERSION_1_1,VK_KHR_sampler_ycbcr_conversion[]
+ifdef::VK_EXT_image_drm_format_modifier[]
+  * ename:VK_IMAGE_ASPECT_MEMORY_PLANE_0_BIT_EXT specifies the
+    _memory plane_ 0.
```

Comment:
I am not quite sure what this does. Also "memory plane" seems to be missing in Glossary.